src/EditorFeatures/TestUtilities/InProcRemoteHostClientProvider.cs

```diff
@@ -18,6 +18,15 @@
 using Roslyn.Test.Utilities.Remote;
 using Roslyn.Utilities;
 
+namespace Microsoft.CodeAnalysis.Remote.Testing
```

Comment:
Another pending PR is moving this file, so for now leave it here to avoid merge issues.